### PR TITLE
Improving readability of code

### PR DESCRIPTION
**Pull request by pchakraborty1**
Adding concise readme to supplement the github docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# DPM360
+Repository for Disease Progression Modeling workbench 360
+
+Overview and YouTube demonstration are available [here](https://ibm.github.io/DPM360/). License, Contribution, Publications are also available there.
+
+<figure><center><img src=./docs/resources/png/dpm360v2.png "DPM360" width="300"/></center><figcaption>DPM360 Component View</figcaption></figure>
+
+# Installation Guides
+
+DPM360 components are interoperable but also work as independent tools. They are working on a cloud cluster (DPM360 server side) and on a client server (DPM360 client side), having separate installation procedures. Please see the guides below to install each component.
```
Comment (pchakraborty1) [inline README.md:11]: suggested change: `also works as` -> `can also work as`

Comment (pchakraborty1) [inline README.md]: suggested changes: 

**Change 1**
`also works as` -> `can also work as`

**Change 2**:
`They are working..` -> 
`DPM360 is typically installed over a cluster that sets up a number of interconnected micro-services. We can broadly divide the components into two groups viz (i) components that are concerned with micro service setups and (ii) standalone python packages providing core functional capabilities`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## DPM360 server side
```
Comment (pchakraborty1) [inline README.md]: suggested change:

`DPM360 server side` -> `DPM360 micro-service utilities`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+For the server side, the <b>installer</b> component sets up an OHDSI stack (Atlas, WebAPI, a Postgres Database, and Achilles) into a cloud cluster such as Kubernetes or OpenShift. See [installation guide](https://github.com/IBM/DPM360/blob/main/installer/docs/installer.md) for details. Its [<b>Express Installation Script</b> section](https://github.com/IBM/DPM360/blob/main/installer/docs/installer.md#express-installation-script) provides minimum setup operations. Using this component:
```
Comment (pchakraborty1) [inline README.md]: Suggested change:

change server side to `One of the key micro-service utilties is <b>installer</b> that does..`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- you run a OMOP CDM database using Postgres on your cloud cluster
```
Comment (pchakraborty1) [inline README.md]: remove `you` from the list of comments

File: README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+- you run Atlas, WebAPI and other OHDSI service with the DB
+- you run Model Registry using MLFlow where your learned models are registered
+
+The <b>service builder</b> component packages and deploys the learned models to the target cloud cluster. See [installation guide](https://github.com/IBM/DPM360/blob/main/service_builder/docs/README.md) for details. Using this component:
+- you make a microservice by deploying the model registered in Model Registry using KFServing
```
Comment (pchakraborty1) [inline README.md]: Same comment as above. Drop `you` from the list

File: README.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+- you test and interact with the deployed model microservice via a Swagger based interface
+
+## DPM360 client side
```
Comment (pchakraborty1) [inline README.md]: Suggested change:

`DPM360 client side` -> `DPM360 standalone python packages`

File: README.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+The [<b>lightsaber</b>](https://github.com/IBM/DPM360/blob/main/lightsaber/docs/index.md) component is an extensible training framework which provides blueprints for the development of disease progression models. See [installation guide](https://github.com/IBM/DPM360/blob/main/lightsaber/docs/install.md). Also see [user guide](https://github.com/IBM/DPM360/blob/main/lightsaber/docs/user_guide.md) for data loading and training details. Using this component:
+- you develop machine learning models using extensible data loaders and training pipelines
```
Comment (pchakraborty1) [inline README.md]: Drop `you` from the list of comments

File: README.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+- you use extensible data loaders designed for time-series dataset extracted from OHDSI and other EMRs
+- you use scikit-learn and PyTorch Lightning based training pipelines with pre-defined networks and loss functions for processing time-series dataset.
+- you save and register your learned models with experimental artifacts in Model Registry
+
+The [<b>cohort tools</b>](https://github.com/IBM/DPM360/blob/main/cohort_tools/docs/index.md) component provides python scripts to extract features from cohorts defined via ATLAS or custom queries. It enables [integration with lightsaber](https://github.com/IBM/DPM360/blob/main/cohort_tools/docs/user_guide.md) to use features extracted from OHDSI databases. Using this component:
+- with cohort defined by Atlas and others, you define and extract the features from the OMOP CDM database
+- you extract such features in CVS files and use them as inputs for lightsaber
```
Comment (pchakraborty1) [inline README.md]: Drop `you` from the list

**Review by pchakraborty1 — COMMENTED** [review]
@akoseki2 please consider the changes as listed out in the review

File: README.md [hunk 9]
```diff
@@ -8,27 +8,27 @@ Overview and YouTube demonstration are available [here](https://ibm.github.io/DP
...
-- you run a OMOP CDM database using Postgres on your cloud cluster
-- you run Atlas, WebAPI and other OHDSI service with the DB
-- you run Model Registry using MLFlow where your learned models are registered
+One of the key micro-service utilties is <b>installer</b> that sets up an OHDSI stack (Atlas, WebAPI, a Postgres Database, and Achilles) into a cloud cluster such as Kubernetes or OpenShift. See [installation guide](https://github.com/IBM/DPM360/blob/main/installer/docs/installer.md) for details. Its [<b>Express Installation Script</b> section](https://github.com/IBM/DPM360/blob/main/installer/docs/installer.md#express-installation-script) provides minimum setup operations. Using this component, you do:
```
Comment (pchakraborty1) [inline README.md]: remove `you do`

File: README.md [hunk 10]
```diff
@@ -8,27 +8,27 @@ Overview and YouTube demonstration are available [here](https://ibm.github.io/DP
...
+- run a OMOP CDM database using Postgres on your cloud cluster
+- run Atlas, WebAPI and other OHDSI service with the DB
+- run Model Registry using MLFlow where your learned models are registered
 
-The <b>service builder</b> component packages and deploys the learned models to the target cloud cluster. See [installation guide](https://github.com/IBM/DPM360/blob/main/service_builder/docs/README.md) for details. Using this component:
-- you make a microservice by deploying the model registered in Model Registry using KFServing
-- you test and interact with the deployed model microservice via a Swagger based interface
+The <b>service builder</b> component packages and deploys the learned models to the target cloud cluster. See [installation guide](https://github.com/IBM/DPM360/blob/main/service_builder/docs/README.md) for details. Using this component, you do:
```
Comment (pchakraborty1) [inline README.md]: remove `you do`

File: README.md [hunk 11]
```diff
@@ -8,27 +8,27 @@ Overview and YouTube demonstration are available [here](https://ibm.github.io/DP
...
-- you develop machine learning models using extensible data loaders and training pipelines
-- you use extensible data loaders designed for time-series dataset extracted from OHDSI and other EMRs
-- you use scikit-learn and PyTorch Lightning based training pipelines with pre-defined networks and loss functions for processing time-series dataset.
-- you save and register your learned models with experimental artifacts in Model Registry
+The [<b>lightsaber</b>](https://github.com/IBM/DPM360/blob/main/lightsaber/docs/index.md) component is an extensible training framework which provides blueprints for the development of disease progression models. See [installation guide](https://github.com/IBM/DPM360/blob/main/lightsaber/docs/install.md). Also see [user guide](https://github.com/IBM/DPM360/blob/main/lightsaber/docs/user_guide.md) for data loading and training details. Using this component, you do:
```
Comment (pchakraborty1) [inline README.md]: remove `you do`

File: README.md [hunk 12]
```diff
@@ -8,27 +8,27 @@ Overview and YouTube demonstration are available [here](https://ibm.github.io/DP
...
+- use scikit-learn and PyTorch Lightning based training pipelines with pre-defined networks and loss functions for processing time-series dataset.
+- save and register your learned models with experimental artifacts in Model Registry
 
-The [<b>cohort tools</b>](https://github.com/IBM/DPM360/blob/main/cohort_tools/docs/index.md) component provides python scripts to extract features from cohorts defined via ATLAS or custom queries. It enables [integration with lightsaber](https://github.com/IBM/DPM360/blob/main/cohort_tools/docs/user_guide.md) to use features extracted from OHDSI databases. Using this component:
-- with cohort defined by Atlas and others, you define and extract the features from the OMOP CDM database
-- you extract such features in CVS files and use them as inputs for lightsaber 
+The [<b>cohort tools</b>](https://github.com/IBM/DPM360/blob/main/cohort_tools/docs/index.md) component provides python scripts to extract features from cohorts defined via ATLAS or custom queries. It enables [integration with lightsaber](https://github.com/IBM/DPM360/blob/main/cohort_tools/docs/user_guide.md) to use features extracted from OHDSI databases. Using this component, you do:
```
Comment (pchakraborty1) [inline README.md]: remove `you do`

**Review by pchakraborty1 — COMMENTED** [review]
Once you address the changes, feel free to merge the PR

**Review by akoseki2 — APPROVED** [review]

**Pull request merged**